skbeam/core/tests/test_correlation.py

```diff
@@ -88,13 +101,43 @@ def test_lazy_multi_tau():
     assert np.average(full_result.g2-1) < 0.01
 
     # run the correlation on the first half
-    gen_first_half = lazy_multi_tau(
+    gen_first_half = lazy_two_time(rois, img_stack[:stack_size//2], stack_size,
+                                   num_bufs=stack_size, num_levels=1)
```

Comment:
YEs `num_frames` can be longer then you will get like following , not enough images to fill,

![screen shot 2016-01-14 at 10 18 57 am](https://cloud.githubusercontent.com/assets/5701652/12328196/4cee6f6a-baa8-11e5-9cc8-9ac54c7bbd5d.png)